grails-app/controllers/org/pih/warehouse/shipping/ShipmentWorkflowController.groovy

```diff
@@ -108,6 +108,7 @@ class ShipmentWorkflowController {
 
     List getDocumentTemplates() {
         def documentTypes = DocumentType.findAllByDocumentCode(DocumentCode.SHIPPING_TEMPLATE)
+        documentTypes += DocumentType.findAllByDocumentCode(DocumentCode.INVOICE_TEMPLATE)
```

Comment:
Use `findAllByDocumentCodeInList` here, instead of fetching twice.

Reply:
done